data/json/scenarios.json

```
@@ -393,6 +393,17 @@
     ],
     "flags": [ "LONE_START", "CHALLENGE", "OUTDOOR" ]
   },
+  {
+    "type": "scenario",
+    "ident": "migo_prisoner",
+    "name": "Challenge - Mi-Go Camp",
+    "points": -8,
+    "description": "You find yourself in the most alien place you've ever seen, the hot humid air and the organic aspect of the structure makes you feel like you're trapped inside a giant creature.  Whatever it is, you need to get out of here before They find you.",
```

```suggestion
    "description": "You find yourself in the most alien place you've ever seen.  the hot humid air and the organic aspect of the structure makes you feel like you're trapped inside a giant creature.  Whatever it is, you need to get out of here before They find you.",
```
A sentence break in here makes sense.